tox.ini

```diff
@@ -5,8 +5,10 @@ envlist =
     pypy
 
 [testenv]
-commands = py.test -q  --doctest-modules
-deps = pytest
+commands = py.test
```

Comment:
I wonder if doctests are still running.

Maybe you should create an error in a doctest to ensure that they are still running

Reply:
> I wonder if doctests are still running.

They do. I've tested it locally. The options from the `setup.cfg` file are passed to pytest. As a summary, I get:

```
126 passed in 0.47 seconds
```

Reply:
According to the [pytest documentation](https://docs.pytest.org/en/latest/customize.html#finding-the-rootdir):

 > "[it] look[s] for pytest.ini, tox.ini and setup.cfg files in the ancestor directory and upwards. If one is matched, it becomes the ini-file and its directory becomes the rootdir."